www/class/centreonGraphNg.class.php

```diff
@@ -1004,10 +1086,23 @@ private function formatByMetrics($rrdData)
                     $metric['prints'][] = array_values($rrdData['meta']['gprints'][$gprintsPos]);
                 }
             }
-            
+
+            $minimum_value = null;
+            $maximum_value = null;
             for ($j = 0; $j < $size; $j++) {
                 $metric['data'][] = $rrdData['data'][$j][$i];
+                if (!is_null($rrdData['data'][$j][$i]) && 
+                    (is_null($minimum_value) || $rrdData['data'][$j][$i] < $minimum_value)) {
```

Comment:
`$rrdData['data'][$j][$i]`

feel like we are mixing up the how and the what
+
there is redondancy with the following tests & assignments (violates DRY)

=

assign the rrdData current index in explicit variable telling the what